deepliif/util/__init__.py

```diff
@@ -26,31 +26,6 @@ def chunker(iterable, size):
         yield iterable[i::size]
 
 
-def tensor2im(input_image, imtype=np.uint8):
```

Comment:
@ricdodds 🤔 if you run `deepliif train` do you get the same error complaining about util.tensor2im cannot be found? I was testing `python cli.py train`, and not sure if it would lead to a slightly different behavior